controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -426,13 +407,66 @@ func (r *ROSAControlPlaneReconciler) rosaClusterToROSAControlPlane(log *logger.L
 	}
 }
 
-func (r *ROSAControlPlaneReconciler) getOcmCluster(rosaScope *scope.ROSAControlPlaneScope, ocmConnection *sdk.Connection) (*cmv1.Cluster, error) {
-	clusterKey := rosaScope.ControlPlane.Name
+// OCMClient is a temporary helper to talk to OCM API.
+// TODO(alberto): vendor this from https://github.com/openshift/rosa/tree/master/pkg/ocm or build its own package here.
+type OCMClient struct {
+	ocm *sdk.Connection
+}
+
+func newOCMClient() (*OCMClient, error) {
+	// Create the connection, and remember to close it:
+	token := os.Getenv("OCM_TOKEN")
```

Comment:
I think we'd be better of reading this kind of thing at startup and passing content down the stack normally, as opposed to doing something like this in helpers.